packages/dashboard/src/lib/components/modals/ProjectModal.svelte

```diff
@@ -1,21 +1,91 @@
 <script lang="ts">
-	import { Modal, Form, FormGroup, TextInput, Checkbox } from 'carbon-components-svelte';
+	import {
+		Modal,
+		Form,
+		FormGroup,
+		TextInput,
+		Checkbox,
+		Select,
+		SelectItem
+	} from 'carbon-components-svelte';
+	import { database } from '$lib/services/database';
+	import { DatabaseResponse } from '$lib/types/databaseResponse';
+	import type { definitions } from '@inlang/database';
+	import { projectStore } from '$lib/stores/projectStore';
+	import { page } from '$app/stores';
+	import { onMount } from 'svelte';
+	import ISO6391 from 'iso-639-1';
 
 	export let open = false;
-	export let primaryButtonDisabled = false;
+	//export let primaryButtonDisabled = false;
+	let isLoading = false;
 
 	export let heading = 'Add project';
 
 	export let projectName: string | '' = '';
+	export let organizationId: string | '' = '';
+	//export let iso_code: string | '' = '';
+
+	let organizations: DatabaseResponse<definitions['organization'][]>;
+	let languageIso: definitions['language']['iso_code'];
+
+	function makeApiKey(length: number) {
+		var result = '';
+		var characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-';
```

Comment:
We need to make sure to not forget invalidating all API keys before the functionality is implemented.